blackjax/sgmcmc.py

```diff
@@ -0,0 +1,17 @@
+from jax import random
```

Comment:
I am going to start the review at the user-interface as it is fairly easy to define. I don't know if you've noticed but the namespace in blackjax is so you can import kernels as `blackjax.algorithm_name`. I would thus call this file `sgld.py`. And `build_SGLD_kernel` would become `kernel`.